default.nix

```diff
@@ -31,6 +32,13 @@ let
       kill $TAILPID
     '';
   });
+  applyBenchmarks = drv: overrideCabal drv (drv: {
+    configureFlags = (drv.configureFlags or []) ++ [
+      "--enable-benchmarks"
+    ];
+    doBenchmark = true;
```

Comment:
I think this could use a comment. Does `doBenchmark = true` not imply the `--enable-benchmarks` flag?